### ISSUE TITLE
Better graphing

### ISSUE THREAD

**Issue by oluoluoxenfree** [RoboPython/transcode#7]
I'll try and roll my own this time rather than being entirely lazy!

**Comment by JessFairbairn** [comment]
What do you have in mind?

**Comment by oluoluoxenfree** [comment]
Very dependant on whether we have more information, but it would be cool to have tooltips, better colours and a more dynamic graph that didn't require a page refresh? 

Probably very much a stretch goal at the minute :P

**Comment by RubyOffThe** [comment]
definitly do able though

**Comment by aychedee** [comment]
With free hosting it's not that easy to setup pushing from the backend to the frontend. But the frontend could always poll the backend every couple of seconds. So we would need a json api for the frontend to poll?

**Comment by oluoluoxenfree** [comment]
Yeah that's what i'm used to using in any case, not sure about what's best practice though!
